docs/api-reference/uiSchema.md

```diff
@@ -218,5 +234,45 @@ const uiSchema = {
   "ui:title": "Your password"
 };
 ```
+
+## submitButtonOptions
+
+Sometimes it's convenient to change the behavior of the submit button for the form. This is the purpose of the `ui:submitButtonOptions` uiSchema directive:
+
+You can pass any other prop to the submit button if you want, by default, this library will set the following options / props mentioned below for all submit buttons:
+
+### `removed` option
```

Comment:
As per discussion in 2/25 call, let's rename this to:

```suggestion
### `norender` option
```

this is more intuitive to the user and reflects how it's actually implemented (button is not rendered)